pkg/virt-controller/watch/preset.go

```diff
@@ -152,6 +152,9 @@ func (c *VirtualMachinePresetController) initializeVirtualMachine(vm *kubev1.Vir
 		if !success {
 			logger.Object(vm).Warning("Marking VM as failed")
 			vm.Status.Phase = kubev1.Failed
+		} else {
+			logger.Object(vm).Info("Setting default values on VirtualMachine")
```

Comment:
That may deserve a higher verbosity level. Maybe `V(4)`?

Reply:
ok, trying to understand how the logger works...
- both `verbosityLevel` and `currentVerbosityLevel` are `0` or whatever is given with verbosityFlag by default... that means info logs are always logged by default because both values are equal, right? Wouldn't it make more sense to set `verbosityLevel` to 2 or 3 by default?
- when I change the `currentVerbosityLevel` with `V(x)`, it will stay at that level until i call `V(x)` again. Is that on purpose?

Reply:
You have to use `V(X)` to specify the minimum vervosity level which needs to be configured to make the message appear. The minimum level can be set via the `-v` flag. So if you configure the logger with `-v 3` on the commandline, all  messages which have used `V(3)` or below will be logged. In other words, the more logs you want to see the higher you set the verbosity level on the commandline.

Reply:
> when I change the currentVerbosityLevel with V(x), it will stay at that level until i call V(x) again. Is that on purpose?

No. It should only be set for invocations after `V()`, e.g. `V(3).Info(""). That looks like a bug.

So this is the desired behaviour:

```
mylogger := logger.Object(vm).V(3) # sets 3 for mylogger
mylogger.Info("lal") # uses 3 for logging
logger.Object(vm).Info("lala") # should use 0
mylogger.V(5).Info("blub") # should use 5
mylogger.Info("lal") # uses still 3
```

Reply:
When looking into the sourcecode, we basically want a copy of the struct but right now it returns a pointer to it.

Reply:
Ok, will add the V(4) in a second.
I will have a look into the copy/pointer topic tomorrow and create an issue if needed.
I still think that the initialization of `verbosityLevel` and `currentVerbosityLevel` could be improved... will also create an issue about it and explain.
Thx for reviewing!